[2024-02-21T07:38:08.153+0000] {taskinstance.py:1956} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: extract_email.get_mail scheduled__2024-02-07T00:00:00+00:00 [queued]>
[2024-02-21T07:38:08.170+0000] {taskinstance.py:1956} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: extract_email.get_mail scheduled__2024-02-07T00:00:00+00:00 [queued]>
[2024-02-21T07:38:08.170+0000] {taskinstance.py:2170} INFO - Starting attempt 1 of 1
[2024-02-21T07:38:08.188+0000] {taskinstance.py:2191} INFO - Executing <Task(PythonOperator): get_mail> on 2024-02-07 00:00:00+00:00
[2024-02-21T07:38:08.195+0000] {standard_task_runner.py:60} INFO - Started process 113 to run task
[2024-02-21T07:38:08.198+0000] {standard_task_runner.py:87} INFO - Running: ['***', 'tasks', 'run', 'extract_email', 'get_mail', 'scheduled__2024-02-07T00:00:00+00:00', '--job-id', '79', '--raw', '--subdir', 'DAGS_FOLDER/extract_email.py', '--cfg-path', '/tmp/tmp_rez3axi']
[2024-02-21T07:38:08.201+0000] {standard_task_runner.py:88} INFO - Job 79: Subtask get_mail
[2024-02-21T07:38:08.250+0000] {task_command.py:423} INFO - Running <TaskInstance: extract_email.get_mail scheduled__2024-02-07T00:00:00+00:00 [running]> on host 893865381b0d
[2024-02-21T07:38:08.353+0000] {taskinstance.py:2480} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='extract_email' AIRFLOW_CTX_TASK_ID='get_mail' AIRFLOW_CTX_EXECUTION_DATE='2024-02-07T00:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2024-02-07T00:00:00+00:00'
[2024-02-21T07:38:08.354+0000] {taskinstance.py:2698} ERROR - Task failed with exception
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/taskinstance.py", line 433, in _execute_task
    result = execute_callable(context=context, **execute_callable_kwargs)
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/operators/python.py", line 196, in execute
    context_merge(context, self.op_kwargs, templates_dict=self.templates_dict)
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/utils/context.py", line 261, in context_merge
    context.update(*args, **kwargs)
  File "/usr/local/lib/python3.8/_collections_abc.py", line 837, in update
    for key, value in other:
ValueError: too many values to unpack (expected 2)
[2024-02-21T07:38:08.360+0000] {taskinstance.py:1138} INFO - Marking task as FAILED. dag_id=extract_email, task_id=get_mail, execution_date=20240207T000000, start_date=20240221T073808, end_date=20240221T073808
[2024-02-21T07:38:08.374+0000] {standard_task_runner.py:107} ERROR - Failed to execute job 79 for task get_mail (too many values to unpack (expected 2); 113)
[2024-02-21T07:38:08.416+0000] {local_task_job_runner.py:234} INFO - Task exited with return code 1
[2024-02-21T07:38:08.427+0000] {taskinstance.py:3280} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2024-02-21T07:59:47.904+0000] {taskinstance.py:1956} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: extract_email.get_mail scheduled__2024-02-07T00:00:00+00:00 [queued]>
[2024-02-21T07:59:47.925+0000] {taskinstance.py:1956} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: extract_email.get_mail scheduled__2024-02-07T00:00:00+00:00 [queued]>
[2024-02-21T07:59:47.926+0000] {taskinstance.py:2170} INFO - Starting attempt 1 of 1
[2024-02-21T07:59:47.948+0000] {taskinstance.py:2191} INFO - Executing <Task(PythonOperator): get_mail> on 2024-02-07 00:00:00+00:00
[2024-02-21T07:59:47.959+0000] {standard_task_runner.py:60} INFO - Started process 383 to run task
[2024-02-21T07:59:47.965+0000] {standard_task_runner.py:87} INFO - Running: ['***', 'tasks', 'run', 'extract_email', 'get_mail', 'scheduled__2024-02-07T00:00:00+00:00', '--job-id', '133', '--raw', '--subdir', 'DAGS_FOLDER/extract_email.py', '--cfg-path', '/tmp/tmp7jlmfznw']
[2024-02-21T07:59:47.968+0000] {standard_task_runner.py:88} INFO - Job 133: Subtask get_mail
[2024-02-21T07:59:48.042+0000] {task_command.py:423} INFO - Running <TaskInstance: extract_email.get_mail scheduled__2024-02-07T00:00:00+00:00 [running]> on host 9bab6a0db245
[2024-02-21T07:59:48.193+0000] {taskinstance.py:2480} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='extract_email' AIRFLOW_CTX_TASK_ID='get_mail' AIRFLOW_CTX_EXECUTION_DATE='2024-02-07T00:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2024-02-07T00:00:00+00:00'
[2024-02-21T07:59:50.584+0000] {ImapSmtp.py:839} INFO - IMAP search: 'gmail:'Message from RPA Python' after:15-12-2023'
[2024-02-21T07:59:53.497+0000] {logging_mixin.py:188} INFO - Message from RPA Python0
[2024-02-21T07:59:53.498+0000] {ImapSmtp.py:1240} INFO - Attachment filepath: PosixPath('/data/attachments/test-182.txt')
[2024-02-21T07:59:53.498+0000] {ImapSmtp.py:1243} INFO - Saving attachment: test.txt
[2024-02-21T07:59:53.498+0000] {logging_mixin.py:188} INFO - Message from RPA Python1
[2024-02-21T07:59:53.499+0000] {ImapSmtp.py:1240} INFO - Attachment filepath: PosixPath('/data/attachments/test-183.txt')
[2024-02-21T07:59:53.499+0000] {ImapSmtp.py:1243} INFO - Saving attachment: test.txt
[2024-02-21T07:59:53.500+0000] {logging_mixin.py:188} INFO - Message from RPA Python2
[2024-02-21T07:59:53.500+0000] {ImapSmtp.py:1240} INFO - Attachment filepath: PosixPath('/data/attachments/test-184.txt')
[2024-02-21T07:59:53.500+0000] {ImapSmtp.py:1243} INFO - Saving attachment: test.txt
[2024-02-21T07:59:53.501+0000] {logging_mixin.py:188} INFO - Message from RPA Python3
[2024-02-21T07:59:53.501+0000] {ImapSmtp.py:1240} INFO - Attachment filepath: PosixPath('/data/attachments/test-185.txt')
[2024-02-21T07:59:53.501+0000] {ImapSmtp.py:1243} INFO - Saving attachment: test.txt
[2024-02-21T07:59:53.502+0000] {logging_mixin.py:188} INFO - Message from RPA Python4
[2024-02-21T07:59:53.502+0000] {ImapSmtp.py:1240} INFO - Attachment filepath: PosixPath('/data/attachments/test-186.txt')
[2024-02-21T07:59:53.502+0000] {ImapSmtp.py:1243} INFO - Saving attachment: test.txt
[2024-02-21T07:59:53.503+0000] {logging_mixin.py:188} INFO - Message from RPA Python5
[2024-02-21T07:59:53.503+0000] {ImapSmtp.py:1240} INFO - Attachment filepath: PosixPath('/data/attachments/test-187.txt')
[2024-02-21T07:59:53.504+0000] {ImapSmtp.py:1243} INFO - Saving attachment: test.txt
[2024-02-21T07:59:53.504+0000] {logging_mixin.py:188} INFO - Message from RPA Python6
[2024-02-21T07:59:53.504+0000] {ImapSmtp.py:1240} INFO - Attachment filepath: PosixPath('/data/attachments/test-188.txt')
[2024-02-21T07:59:53.505+0000] {ImapSmtp.py:1243} INFO - Saving attachment: test.txt
[2024-02-21T07:59:53.505+0000] {logging_mixin.py:188} INFO - Message from RPA Python7
[2024-02-21T07:59:53.506+0000] {ImapSmtp.py:1240} INFO - Attachment filepath: PosixPath('/data/attachments/test-189.txt')
[2024-02-21T07:59:53.506+0000] {ImapSmtp.py:1243} INFO - Saving attachment: test.txt
[2024-02-21T07:59:53.506+0000] {logging_mixin.py:188} INFO - Message from RPA Python8
[2024-02-21T07:59:53.507+0000] {ImapSmtp.py:1240} INFO - Attachment filepath: PosixPath('/data/attachments/test-190.txt')
[2024-02-21T07:59:53.507+0000] {ImapSmtp.py:1243} INFO - Saving attachment: test.txt
[2024-02-21T07:59:53.507+0000] {logging_mixin.py:188} INFO - Message from RPA Python9
[2024-02-21T07:59:53.508+0000] {ImapSmtp.py:1240} INFO - Attachment filepath: PosixPath('/data/attachments/test-191.txt')
[2024-02-21T07:59:53.508+0000] {ImapSmtp.py:1243} INFO - Saving attachment: test.txt
[2024-02-21T07:59:54.090+0000] {python.py:201} INFO - Done. Returned value was: None
[2024-02-21T07:59:54.103+0000] {taskinstance.py:1138} INFO - Marking task as SUCCESS. dag_id=extract_email, task_id=get_mail, execution_date=20240207T000000, start_date=20240221T075947, end_date=20240221T075954
[2024-02-21T07:59:54.122+0000] {local_task_job_runner.py:234} INFO - Task exited with return code 0
[2024-02-21T07:59:54.135+0000] {taskinstance.py:3280} INFO - 0 downstream tasks scheduled from follow-on schedule check
